R/estimate_and_apply_delay.R

```diff
@@ -0,0 +1,68 @@
+#' Estimate and apply delay from a reporting triangle
+#'
+#' @inheritParams estimate_delay
+#' @param ... Additional arguments
```

Comment:
For what?

Reply:
This was requested in previous round of revisions so that we pass the default `n` from `estimate_delay()`.

Reply:
no I mean how does the user know what this does as it isn't documented at all where the ... goes?

Reply:
Ah I see, have fixed to document